Sming/SmingCore/Network/MqttClient.h

```diff
@@ -54,6 +60,7 @@ class MqttClient: protected TcpClient
 	uint8_t *current;
 	int posHeader;
 	MqttStringSubscriptionCallback callback;
+	int keepAlive = 50;
```

Comment:
You are right. I will set it back to 20 and then the developer can decide how often the ping should happen.